src/ucp/core/ucp_context.c

```diff
@@ -428,8 +430,55 @@ ucs_status_t ucp_config_read(const char *env_prefix, const char *filename,
     return status;
 }
 
+static void ucp_cached_key_release(ucs_config_cached_key_t *key_val)
+{
+    if (key_val == NULL) {
+        return;
+    }
```

Comment:
how can it be NULL?

Reply:
It's defensive coding. Should I remove it?

Reply:
@yosefe remove the ```check``` here.

Reply:
can convert to assert

Reply:
Done. use "ucs_assert"